tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -435,6 +435,86 @@ test.describe( 'Nested Tabs tests', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Check that active tab receives custom hover color when hovering above it', async ( { page }, testInfo ) => {
```

Comment:
Suggestion: 'Check if the hover effect works on active tabs`

Reply:
Great suggestion.
Done.